requirements/test.txt

```diff
@@ -32,10 +32,6 @@ distlib==0.3.8
     # via
     #   -r requirements/dev.txt
     #   virtualenv
-django==4.2.15
```

Comment:
Django was added due to `pylint-django` package.

Reply:
Although `edx-lint` is a linting tool and doesn't require Django itself, the django package is included in `test.txt` due to the use of `pylint-django`. To maintain consistency with the existing pattern, we're adding support for both Django 4.2 and 5.2, reflecting the presence of the django dependency in the testing configuration.